lab/SoundTest/SoundTest.js

```diff
@@ -36,7 +36,7 @@ const SOUND_MANIFEST = {
     layered_melody: layered_melody(),
     phase_scale: phase_scale(),
     symmetry_melody: symmetry_melody(),
-    binary_progression: binary_chords(),
+    //binary_progression: binary_chords(),
```

Comment:
Make sure to debug this one! I was getting an empty array somehow, see what I did wrong.